frontend/src/static/js/utils/repo_sorter.js

```diff
@@ -0,0 +1,134 @@
+function getTotalCommits(total, group) {
+  return total + group.checkedFileTypeContribution;
+}
+
+function getGroupCommitsVariance(total, group) {
+  return total + group.variance;
+}
+
+// function getGroupCommitsVariance(sortingOption) {
+//   return function (total, group) {
+//     if (sortingOption === 'totalCommits') {
+//       return total + group.checkedFileTypeContribution;
+//     }
+//     return total + group[sortingOption];
+//   };
+// }
+
+function sortingHelper(element, sortingOption) {
+  // return sortingOption === 'totalCommits' || sortingOption === 'variance'
+  //     ? element.reduce(getGroupCommitsVariance, 0)
+  //     : element[0][sortingOption];
+  if (sortingOption === 'totalCommits') {
+    return element.reduce(getTotalCommits, 0);
+  }
+  if (sortingOption === 'variance') {
+    return element.reduce(getGroupCommitsVariance, 0);
+  }
+  return element[0][sortingOption];
+}
```

Comment:
The change I made to the code. Will remove comment once verified is ok.

Reply:
Any reason for making this change?

Reply:
Actually I realized that the commented code was not the original code sorry.

The reason why I changed the original code is because it was using `this.sortingOption` which is not available when extracted, and that returning an anonymous function is not the best idea (I think there was a warning).